dev_tools/utils/nxdl_utils.py

```diff
@@ -363,7 +369,12 @@ def find_definition_file(bc_name):
     Note that it first checks in contributed and goes beyond only if no contributed found
     """
     bc_filename = None
-    for nxdl_folder in ["contributed_definitions", "base_classes", "applications"]:
+    for nxdl_folder in [
```

Comment:
It seems a bit awkward to also search the `"dev_tools/tests/data"` folder here outside of the tests. Is it somehow possible to add this only in the test stage?